src/harness/reference_models/ppa_gwpz_fss_gwbl_purge/ppa_gwpz_fss_gwbl_purge.py

```diff
@@ -0,0 +1,86 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
```

Comment:
Can we move this file in a more sensible existing directory, for example: "purge". Or does it belong logically to the pre_iap_filtering directory: I see there is a cyclical import from these 2 modules. We import here something from pre_iap_filtering, and there we import this file.

Also there is another PR #462  about FSS purge list, which should also be put in the same directory ?

Also can we change the file name into: "purge.py" for example ?

Reply:
renamed   file to 'zone_purge.py' as suggested and moved to 'pre_iap_filtering'   directoy.          For FSS purge list, we will move the files in #PR462